<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>835</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #282828</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>40</y>
      <width>1131</width>
      <height>771</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
    </layout>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>1366</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #535353</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>1920</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(83, 83, 83)</string>
    </property>
    <widget class="QToolButton" name="add_node_btn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>6</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;Add nodes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Add nodes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">#add_node_btn{
background-color: transparent;
border-image: url(&quot;resource/gui/tool/add.png&quot;);
background: none;
border-radius: 3px;
background-repeat: none;
}
#add_node_btn:hover{
background-color: #454545;
border-radius: 3px;
border-image: url(&quot;resource/gui/tool/add.png&quot;);
}
#add_node_btn:pressed{
background-color: #383838;
border-radius: 3px;
border-image: url(&quot;resource/gui/tool/add.png&quot;);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="zoom_in_btn">
     <property name="geometry">
      <rect>
       <x>93</x>
       <y>6</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body style=&quot;background-color: #dddddd; border-radius: 5px;&quot;&gt;&lt;img src=&quot;resource/gui/tooltip/zoom-in.png&quot;&gt;&lt;p&gt;&lt;strong&gt;Zoom In&lt;/strong&gt;&lt;br/&gt;Magnifies or reduces the view of an image&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Add nodes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">#zoom_in_btn{
background-color: transparent;
border-image: url(&quot;resource/gui/tool/zoom-in.png&quot;);
background: none;
border-radius: 3px;
background-repeat: none;
}
#zoom_in_btn:hover{
background-color: #454545;
border-radius: 3px;
border-image: url(&quot;resource/gui/tool/zoom-in.png&quot;);
}
#zoom_in_btn:pressed{
background-color: #383838;
border-radius: 3px;
border-image: url(&quot;resource/gui/tool/zoom-in.png&quot;);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="zoom_out_btn">
     <property name="geometry">
      <rect>
       <x>136</x>
       <y>6</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;Zoom out&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Add nodes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">#zoom_out_btn{
background-color: transparent;
border-image: url(&quot;resource/gui/tool/zoom-out.png&quot;);
background: none;
border-radius: 3px;
background-repeat: none;
}
#zoom_out_btn:hover{
background-color: #454545;
border-radius: 3px;
border-image: url(&quot;resource/gui/tool/zoom-out.png&quot;);
}
#zoom_out_btn:pressed{
background-color: #383838;
border-radius: 3px;
border-image: url(&quot;resource/gui/tool/zoom-out.png&quot;);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="zoom_reset_btn">
     <property name="geometry">
      <rect>
       <x>179</x>
       <y>6</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Reset zoom&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Add nodes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">#zoom_reset_btn{
background-color: transparent;
border-image: url(&quot;resource/gui/tool/zoom-reset.png&quot;);
background: none;
border-radius: 3px;
background-repeat: none;
}
#zoom_reset_btn:hover{
background-color: #454545;
border-radius: 3px;
border-image: url(&quot;resource/gui/tool/zoom-reset.png&quot;);
}
#zoom_reset_btn:pressed{
background-color: #383838;
border-radius: 3px;
border-image: url(&quot;resource/gui/tool/zoom-reset.png&quot;);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="color_picker_btn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>6</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body style=&quot;background-color: #dddddd; border-radius: 5px;&quot;&gt;&lt;img src=&quot;resource/gui/tooltip/color-picker.png&quot;&gt;&lt;p&gt;&lt;strong&gt;Color picker tool&lt;/strong&gt;&lt;br/&gt;Pick a color or sample colors from an image&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Add nodes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">#color_picker_btn{
background-color: transparent;
border-image: url(&quot;resource/gui/tool/color-picker.png&quot;);
background: none;
border-radius: 3px;
background-repeat: none;
}
#color_picker_btn:hover{
background-color: #454545;
border-radius: 3px;
border-image: url(&quot;resource/gui/tool/color-picker.png&quot;);
}
#color_picker_btn:pressed{
background-color: #383838;
border-radius: 3px;
border-image: url(&quot;resource/gui/tool/color-picker.png&quot;);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="graph_btn">
     <property name="geometry">
      <rect>
       <x>222</x>
       <y>6</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Export graphs&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Add nodes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">#graph_btn{
background-color: transparent;
border-image: url(&quot;resource/gui/tool/graph.png&quot;);
background: none;
border-radius: 3px;
background-repeat: none;
}
#graph_btn:hover{
background-color: #454545;
border-radius: 3px;
border-image: url(&quot;resource/gui/tool/graph.png&quot;);
}
#graph_btn:pressed{
background-color: #383838;
border-radius: 3px;
border-image: url(&quot;resource/gui/tool/graph.png&quot;);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
    <widget class="QComboBox" name="selectLayout">
     <property name="geometry">
      <rect>
       <x>1160</x>
       <y>6</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#selectLayout{
	color: rgb(200, 200, 200);
	border-style: none;
    border-radius: 7px;
	background-color: #383838;
	padding-left: 13px;
}

QComboBox::drop-down 
{
    border: 0px; /* This seems to replace the whole arrow of the combo box */
}
QComboBox::down-arrow {
   image: url(&quot;resource/gui/tool/down-arrow.png&quot;);
    width: 9px;
    height: 9px;
	padding-right: 10px;
}</string>
     </property>
    </widget>
    <widget class="QComboBox" name="selectClusteringAlgo">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>6</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#selectClusteringAlgo{
	color: rgb(200, 200, 200);
	border-style: 2px solid rgb(200, 200, 200);
    border-radius: 7px;
	background-color: #383838;
	padding-left: 13px;
	padding-right: 10px;
}
QComboBox::drop-down 
{
    border: 0px; /* This seems to replace the whole arrow of the combo box */
}
QComboBox::down-arrow {
   image: url(&quot;resource/gui/tool/down-arrow.png&quot;);
    width: 9px;
    height: 9px;
	padding-right: 10px;
}
</string>
     </property>
    </widget>
    <widget class="QToolButton" name="findShortestPathBtn">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>0</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Reset zoom&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Add nodes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">#zoom_reset_btn{
background-color: transparent;
border-image: url(&quot;resource/gui/tool/zoom-reset.png&quot;);
background: none;
border-radius: 3px;
background-repeat: none;
}
#zoom_reset_btn:hover{
background-color: #454545;
border-radius: 3px;
border-image: url(&quot;resource/gui/tool/zoom-reset.png&quot;);
}
#zoom_reset_btn:pressed{
background-color: #383838;
border-radius: 3px;
border-image: url(&quot;resource/gui/tool/zoom-reset.png&quot;);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="editBtn">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>0</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Reset zoom&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Add nodes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">#zoom_reset_btn{
background-color: transparent;
border-image: url(&quot;resource/gui/tool/zoom-reset.png&quot;);
background: none;
border-radius: 3px;
background-repeat: none;
}
#zoom_reset_btn:hover{
background-color: #454545;
border-radius: 3px;
border-image: url(&quot;resource/gui/tool/zoom-reset.png&quot;);
}
#zoom_reset_btn:pressed{
background-color: #383838;
border-radius: 3px;
border-image: url(&quot;resource/gui/tool/zoom-reset.png&quot;);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1366</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #535353</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>40</y>
      <width>251</width>
      <height>771</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(83, 83, 83);
	</string>
    </property>
    <layout class="QVBoxLayout" name="infoArea"/>
   </widget>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>widget</zorder>
   <zorder>line_3</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>line_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #535353; color: #dddddd;
	</string>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="actionSave_Image"/>
    <addaction name="action_Close"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Add_nodes"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionReset_Zoom"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionEdit_graph"/>
    <addaction name="actionFind_shortest_path"/>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="action_Minimize"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout_NetVisual"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Window"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_Image">
   <property name="text">
    <string>Save &amp;Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Add_nodes">
   <property name="text">
    <string>&amp;Add nodes</string>
   </property>
  </action>
  <action name="action_Minimize">
   <property name="text">
    <string>&amp;Minimize</string>
   </property>
  </action>
  <action name="actionAbout_NetVisual">
   <property name="text">
    <string>About &amp;NetVisual</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="text">
    <string>Zoom &amp;In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="text">
    <string>Zoom &amp;Out</string>
   </property>
  </action>
  <action name="actionReset_Zoom">
   <property name="text">
    <string>Reset &amp;Zoom</string>
   </property>
  </action>
  <action name="actionFind_shortest_path">
   <property name="text">
    <string>Find shortest path</string>
   </property>
  </action>
  <action name="actionEdit_graph">
   <property name="text">
    <string>Edit graph</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
